js/components/graph/Graph.js

```diff
@@ -422,6 +422,20 @@ export class Graph extends React.Component {
     }
   }
 
+  nodeUnselect = courseId => {
+    const courseLabelArray = this.state.nodesJSON[courseId].text
+    const courseLabel = courseLabelArray[courseLabelArray.length - 1].text
+    const wasSelected = this.state.nodesStatus[courseId].selected
+    const temp = this.state.selectedNodes
+
+    if (typeof this.props.incrementFCECount === "function" && wasSelected) {
```

Comment:
@mimischly7 in response to your question: this is a common pattern when the prop might be _omitted_ entirely, so it ends up being `undefined`.